sdk/core/azure-core/CMakeLists.txt

```diff
@@ -78,10 +77,11 @@ set(
     inc/azure/core/operation.hpp
     inc/azure/core/operation_status.hpp
     inc/azure/core/platform.hpp
-    inc/azure/core/response.hpp
     inc/azure/core/uuid.hpp
     inc/azure/core/version.hpp
     inc/azure/core.hpp
+    inc/azure/etag.hpp
```

Comment:
Looking at this  bit more closely, `Etag` which is just a plain wrapper on string belonging in the root Azure namespace seems too visible.
I don't think folks would be using this type all that often.

Reply:
But .NET has it under Azure. @tg-msft, @pakrym - what do you think? Should this be at the root `Azure` namespace?

Reply:
It's something customers would construct to make client calls and in .NET these types are in `Azure` namespace.

Reply:
Really? Can you please share a couple of public APIs that accept Etag as input in .NET?
C++ currently only returns it as output, from model types.